File: src/main/java/compiler/block/imports/ImportBlock.java
package compiler.block.imports;

import compiler.block.Block;

public class ImportBlock extends Block {

    public String directory, fileName;

    public ImportBlock(String directory, String fileName) {
        super(null, false, false);
        this.directory = directory;
        this.fileName = fileName;
    }

    public String getName() {
        return null;
    }

    @Override
    public String getValue() {
        return null;
    }

    public String getType() {
        return null;
    }

    @Override
    public String getOpeningCode() {
        return null;
    }

    @Override
    public String getClosingCode() {
        return null;
    }

    @Override
    public String getBodyCode() {
        return null;
    }


    @Override
    public void init() {

    }

}

File: src/main/scala/compiler/block/imports/ImportBlock.scala
package compiler.block.imports

import compiler.block.Block

class ImportBlock(var directory: String, var fileName: String) extends Block(null, false, false) {
  def init() {
  }

  def getName: String = {
    return null
  }

  def getValue: String = {
    return null
  }

  def getType: String = {
    return null
  }

  def getOpeningCode: String = {
    return null
  }

  def getClosingCode: String = {
    return null
  }

  override def toString: String = {
    return "import: " + directory + " : " + fileName
  }
}